bin/oneoff/backfill_data/storage_apps_project_type_and_standalone

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+
+# This script sets the `project_type` and `standalone` columns of the `storage_apps` table, if not
+# already set, based on information from the `value` column.
+
+require File.expand_path('../../../../pegasus/src/env', __FILE__)
+require src_dir 'database'
+
+PROJECT_TYPES = %w(
+  algebra_game
+  applab
+  artist
+  calc
+  eval
+  gamelab
+  makerlab
+  playlab
+  starwars
+  weblab
+).freeze
+
+# @param [String] value_parsed the parsed JSON blob in `pegasus.storage_apps.value`.
+# @return [String | nil] the project type of the storage_app.
+def project_type_from_value(value_parsed)
+  level = value_parsed['level']
+  return nil unless level
+
+  PROJECT_TYPES.each do |type|
+    return type if level.include? type
+  end
+
+  level_id = value_parsed['levelId']
+  PROJECT_TYPES.each do |type|
+    return type if level_id.include? type
+  end
+
+  level_url = value_parsed['levelUrl']
+  PROJECT_TYPES.each do |type|
+    return type if level_url.include? type
+  end
```

Comment:
ok. in that case, it looks like you need to do something different than `return nil unless level`, to only skip over the rest of the checks for `level` but not skip the checks for `levelId` and `levelUrl`.